Neos.FluidAdaptor/Classes/Core/ViewHelper/ViewHelperResolver.php

```diff
@@ -78,6 +84,10 @@ public function initializeObject($reason)
                 $this->addNamespace(strtolower($package->getPackageKey()), $viewHelperNamespace);
             }
         }
+
+        foreach ($this->defaultNamespacesFromConfiguration as $identifier => $namespace) {
```

Comment:
Does this support the syntax we see in `getDefaultNamespaces`, when we get the values form Settings.yaml? From `getDefaultNamespaces` we se one identifier (`f`) and multiple namespaces.

Reply:
Yes, it does. The configuration does get injected as an array (if defined as an array inside Settings.yaml) and addNamespace() does allow arrays as second parameter too. I tried this syntax:
```
Neos:
  FluidAdaptor:
     defaultNamespaces:
       f:
         - 'TYPO3Fluid\Fluid\ViewHelpers'
         - 'Neos\FluidAdaptor\ViewHelpers'
```
This worked fine.